include/ua_client_highlevel.h

```diff
@@ -212,6 +212,67 @@ UA_Client_readUserExecutableAttribute(UA_Client *client, const UA_NodeId nodeId,
                                      &UA_TYPES[UA_TYPES_BOOLEAN]);
 }
 
+/**
+ * Historical Access
+ * ^^^^^^^^^^^^^^^^^
+ * The following functions can be used to read a single node historically.
+ * Use the regular service to read several nodes at once.
+ */
+typedef UA_Boolean
+(*UA_HistoricalIteratorCallback)(const UA_NodeId nodeId, const UA_Boolean isInverse,
```

Comment:
- Add the client pointer as the first argument
- The nodeId can be a const pointer
- Integers don't have to be const
- rename "handle" to "callbackContext"

What is the meaning of moreDataAvailable?
There is a continuation in the server?

What if the client fails after signaling moreDataAvailable?
We could model the "continuation" as a persistent context also on the client side.
And close the "continuation" with a status code in the callback if an error occurs.